packages/twenty-front/src/modules/workflow/search-variables/components/VariableTagInput.tsx

```diff
@@ -201,11 +215,18 @@ export const VariableTagInput = ({
     <StyledContainer>
       {label && <StyledLabel>{label}</StyledLabel>}
       <StyledInputContainer multiline={!!multiline}>
-        <StyledEditor multiline={!!multiline}>
+        <StyledEditor multiline={!!multiline} readonly={!!readonly}>
```

Comment:
set readonly optional so you don't need the `!!`

Reply:
you are right